packages/components/src/components/block/block.tsx

```diff
@@ -163,7 +163,7 @@ export class Block extends LitElement implements InteractiveComponent {
   /**
    * When `true`, expands the component and its contents.
    *
-   * @deprecated Use `expanded` prop instead.
+   * @deprecated in v3.1.0, removal target v6.0 - Use `expanded` prop instead.
```

Comment:
Consistency and copyedit suggestion:
```suggestion
   * @deprecated in v3.1.0, removal target v6.0.0 - Use the `expanded` property instead.
```